henry/commands/pulse.py

```diff
@@ -45,8 +46,8 @@ def check_db_connections(self):
                 models.DelimSequence(connection.dialect.connection_tests),
             )
             results = list(filter(lambda r: r.status == "error", resp))
-            errors = [f"- {fill(cast(str, e.message), width=100)}" for e in results]
-
+            errors = [
+                f"- {fill(cast(str, e.message), width=100)}" for e in results]
```

Comment:
Autoformatting

Reply:
ditto

Reply:
it was the autopep8 module VS Code installed breaking over long lines. Removed now